pkg/eventreason/eventreason.go

```diff
@@ -41,6 +41,9 @@ const (
 	// KEDAScalersStopped is for event when scalers watch was stopped for ScaledObject or ScaledJob
 	KEDAScalersStopped = "KEDAScalersStopped"
 
+	// KEDAScalersPaused is for event when scalers watch was paused for ScaledJob
+	KEDAScalersPaused = "KEDAScalersPaused"
```

Comment:
We already support pausing in Scaled Object, can you ensure we are using the same event type please?